### DISCUSSION TITLE
How to omit the matched content entirely

### DISCUSSION THREAD

**Discussion by fedemengo** [BurntSushi/ripgrep#2417] [General] [answered]
I'm using `rg` as search tool in vim and I would like it to only output matches in the format

```
<filename>:<linenos>:<col>
```
I can do all of it (`--vimgrep` basically takes care of it) besides omitting the matched content. I've looked through the manual but I haven't found a way to do that.

Am I missing something obvious or this is not supported? In case it's not, what's your take on supporting this with an additional flag? Maybe `--no-matched-line`? I could open a PR

**Comment by BurntSushi** [comment, marked as answer]
You could just do `-r ''`, but you'll be left with a trailing colon. I believe that's as close as you can get.

I am not in favor of adding a flag to do anything more for this unless you can make a compelling argument that this is something a ton of folks would benefit from. (I suspect that's not the case.)

**Reply by fedemengo** [reply]
Yeah, I guess it's a very specific use case. I think `-r ''` will do it, thanks!
